cycle_en_terre_website_design/views/website_templates.xml

```diff
@@ -0,0 +1,17 @@
+<openerp>
+   <!--We add the css stylesheet -->
+  <template id="website_frontend" name="Design Cycle en Terre"
+    inherit_id="website.assets_frontend">
+    <xpath expr="." position="inside">
+      <link rel="stylesheet"  type="text/less"
+        href="/cycle_en_terre_website_design/static/src/less/cycle_en_terre_website_design.less"/>
+    </xpath>
+  </template>
+    <!--We add a new DIV above the footer-->
+  <template id="website_layout_background_image_footer" name="Design Cycle en Terre background-image footer"
+    inherit_id="website.footer_custom">
+    <xpath expr="//div[@id='footer']" position="before">
+      <div class="bg-img-footer"></div>
+    </xpath>
+  </template>
+</openerp>
```

Comment:
Line should always terminate with a `end of line` character. This file is missing end of line character at the end of the file.

Reply:
Could you be more specific ? Thanks

Reply:
I will explain this later. For now it's ok like this. :)